src/modularize.js

```diff
@@ -0,0 +1,13 @@
+let fs = require('fs');
+let path = require('path');
+
+module.exports = function(ctor, dir) {
```

Comment:
Good ooo but still digesting -- slowly getting used to expressisms but looks promising